smoketest/graphqlnotifiedalerts_test.go

```diff
@@ -0,0 +1,171 @@
+package smoketest
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/target/goalert/smoketest/harness"
+)
+
+/*
+ * This file tests that the alerts GraphQL query shows the proper amount
+ * of results when flipping between "includeNotified" and "favoritesOnly"
+ * query options.
+ *
+ * Service 1: Notified alert
+ * Service 2: Favorited alert
+ */
+func TestNotifiedAlerts(t *testing.T) {
```

Comment:
Use `// line-style` and start with `TestNotifiedAlerts` to document functions.

Reply:
Changed comment to the style you requested. Also, started it with `TestNotifiedAlerts`.